toolbox/tests/bulk_processing/test_uninvalidate_address_processor.py

```diff
@@ -0,0 +1,52 @@
+from unittest.mock import patch
+
+import pytest
+
+from toolbox.bulk_processing.bulk_processor import BulkProcessor
+from toolbox.bulk_processing.uninvalidate_address_processor import UnInvalidateAddressProcessor
+
+
+@pytest.mark.parametrize('case_id',
+                         [('test_case_id', 'TEST'),
+                          ('anything', 'BLAH'), ])
+def test_build_event_messages(case_id):
+    # Given
+    uninvalidate_address_processor = UnInvalidateAddressProcessor()
+
+    # When
+    event_message = uninvalidate_address_processor.build_event_messages({'CASE_ID': case_id})
+
+    # Then
+    assert len(event_message) == 1, 'Should build one and only 1 invalid address event message'
+    assert event_message[0]['event']['type'] == 'RM_UNINVALIDATE_ADDRESS'
+    assert event_message[0]['event']['transactionId'] is not None
+    assert event_message[0]['event']['dateTime'] is not None
+    assert event_message[0]['payload']['rmUnInvalidateAddress']['caseId'] == case_id
+
+
+@patch('toolbox.bulk_processing.bulk_processor.storage')
```

Comment:
You should be able to get rid of this on all these tests and just pass in `patched_storage` to the test, this patching is already set up as a fixture in `../tests/bulk_processing/conftest.py`